packages/fx-core/resource/package.nls.json

```diff
@@ -705,6 +709,9 @@
   "driver.teamsApp.summary.validate.failed": "%s failed",
   "driver.teamsApp.summary.validate.warning": "%s warning",
   "driver.teamsApp.summary.validate.all": "All",
+  "driver.teamsApp.summary.validateWithTestCases": "Validation request has been executed, status: %s.",
+  "driver.teamsApp.summary.validateWithTestCases.result": "Validation request has been executed, status: %s. View the result from [Teams Developer Portal](%s).",
```

Comment:
Proposed: Validation request completed, status: %s. View the result from [Teams Developer Portal](%s).